### ISSUE TITLE
canonicalize worker returned null — the request body was rejected or its response did not fit the buffer.

### ISSUE THREAD

**Issue by KoenCa** [sergioazoc/oxlint-tailwindcss#130]
Got following message:

> canonicalize worker returned null — the request body was rejected or its response did not fit the buffer.
Hint: This is a bug; please open an issue with the input that triggered it. (oxc tailwindcss(enforce-canonical))

How it was triggered and is reproducible:
1. I had following className string:

```
"px-[calc(var(--input-field-text-area-lg-padding-x)+var(--input-field-text-area-lg-item-padding-x))] py-[calc(var(--input-field-text-area-lg-padding-y)+var(--input-field-text-area-lg-item-padding-y))]"
```

2. I started typing and added another `+var()` and stopped typing to copy paste the variable name in there, but now when I try to reproduce it I see that typing a `+` is enough, so the abouve becomes:

```
px-[calc(var(--input-field-text-area-lg-padding-x)+var(--input-field-text-area-lg-item-padding-x)+)] py-[calc(var(--input-field-text-area-lg-padding-y)+var(--input-field-text-area-lg-item-padding-y))]
```

<img width="725" height="141" alt="Image" src="https://github.com/user-attachments/assets/bfcc3926-1c71-4d84-a31e-446a8272791e" />

Even when I finish the whole expression, the warning still stays. If I restart the extensions in the editor, it is gone.
